### DISCUSSION TITLE
How can I fix "git pull" authentication issues after enabling 2FA on my GitHub account?

### DISCUSSION THREAD

**Discussion by Adamibnualfatahh** [community/community#147603] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone,  
I recently enabled two-factor authentication (2FA) on my GitHub account for added security. However, now when I try to run `git pull` or other Git operations, I get authentication errors. I used to use my GitHub password for these commands, but it seems that no longer works.  

Does anyone know how to fix this issue and authenticate properly after enabling 2FA?  
Thanks in advance!

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @Adamibnualfatahh!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
